README.md

```diff
@@ -1,116 +1,134 @@
-
-* View details of our overal project aim, description, and implementation on the [Wiki](https://github.com/GenEpiO/geem/wiki/) site.
-* A working prototype of the portal is available at <http://genepio.org/geem>. This version doesn't currently incorporate the Django framework yet.
+# The Genomic Epidemiology Entity Mart
+
+The Genomic Epidemiology Entity Mart (GEEM) is a portal for examining and 
+downloading ontology-driven specifications for standardized data components. 
+The portal aims to provide term reviewers and software developers with ways to 
+utilize application ontology contents (a collection of terms and relations 
+from other ontologies that combine to model the operation of some domain) 
+without the need to be trained in ontology curation or querying. Ontology-
+driven standards benefit from features of open-source published OWL 2.0 
+ontologies such as globally unique identifiers for terms, multilingual label 
+and definition functionality, and logical validation and reasoning over 
+controlled vocabularies. Such a specificatio#n can be designed to satisfy the 
+requirements of an environmental pH measurement, or a person's age, or a more 
+structured entity like a contact address, or a genomic sequence repository 
+submission for example. 
+
+* View details of our overal project aim, description, and implementation on 
+  the [Wiki](https://github.com/GenEpiO/geem/wiki/) site.
+* A working prototype of the portal is available at <http://genepio.org/geem>. 
+  This version doesn't currently incorporate the Django framework yet.
 
 ___________
 
-## Creating a Development Server
+## Deployment
 
-### Using Docker
-
-1. Build the GEEM docker image:
+Clone this repository.
 
 ```bash
-$ docker build .
+$ git clone https://github.com/GenEpiO/geem.git
 ```
 
-2. Set up the database:
+### Setting up social authentication
 
-```bash
-$ docker-compose run web python /code/manage.py makemigrations --noinput
-$ docker-compose run web python /code/manage.py migrate --noinput
-```
+To run a local server, create the file `.envs/.local/.auth`. To run a 
+production server, create the file `.envs/.production/.auth`. Neither of these 
+files are tracked by Github.
 
-3. Create your admin user:
+Templates of both files are found at `.envs/.local/.auth.template` and 
+`.envs/.production/.auth.template`.
 
-```bash
-$ docker-compose run web python /code/manage.py createsuperuser
-```
+#### SOCIAL_AUTH_GITHUB_KEY and SOCIAL_AUTH_GITHUB_SECRET
 
-4. Load initial data:
+[Register an Oauth2 application on Github.][1] If you are running a local 
+server, enter `http://localhost:8000/index.html` as Homepage URL, and 
+`http://localhost:8000/complete/github/` as Authorization callback URL. If 
+you are running a production server, enter 
+`http://localhost:8888/geem/index.html` as Homepage URL, and 
+`http://localhost:8888/geem/complete/github/` as Authorization callback URL.
```

Comment:
Is this still true about the 8888 port?  Does that affect oauth?

Reply:
It is still true. Watson has it's own extra NGINX container that routes traffic from port 80 to 8888. If you run it on your own machine at localhost, it will be port 8888.